### PR TITLE
github-cli: update to 2.19.0

### PR DESCRIPTION
**Pull request by tranzystorek-io**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/github-cli/template [hunk 1]
```diff
@@ -29,9 +29,9 @@ post_install() {
 	CGO_ENABLED=0 GOARCH="" go run script/build.go manpages
 	vcopy share/man usr/share
 
-	GH="${DESTDIR}/usr/bin/gh"
+	local gh="${DESTDIR}/usr/bin/gh"
```
Comment (paper42) [inline srcpkgs/github-cli/template:32]: from what I can see, this variable is only used once, so I don't think it makes much sense

Comment (tranzystorek-io) [reply srcpkgs/github-cli/template:32]: But it doesn't hurt to wrap this variable into local, even just to show that it's a one-off

Comment (paper42) [reply srcpkgs/github-cli/template:32]: It doesn't hurt, but since we are already improving it, we can get rid of it.

Comment (tranzystorek-io) [reply srcpkgs/github-cli/template]: Ah, I misunderstood what you requested.

**Pull request merged**